lib/phoenix_storybook/views/layout_view.ex

```diff
@@ -64,6 +65,14 @@ defmodule PhoenixStorybook.LayoutView do
 
   defp title(conn_or_socket), do: storybook_setting(conn_or_socket, :title, "Live Storybook")
 
+  defp color_mode_icons(conn_or_socket) do
+    storybook_setting(conn_or_socket, :color_mode_icons,
+      light: {:fa, "brightness", :regular},
+      dark: {:fa, "moon", :regular},
+      system: {:fa, "circle-half-stroke", :regular}
+    )
+  end
```

Comment:
Defaults to original `:fa` icons. Can elevate to mod attr, application env, etc. if desired